samples/blink/main.cpp

```diff
@@ -35,3 +35,7 @@ int main() {
 
     return 0;
 }
+
+
```

Comment:
NIT: There should only be 1 newline at the end of the file. I think this will be fixed automatically when you get the GitHub action running